src/XMakeTasks/UnitTests/Copy_Tests.cs

```diff
@@ -2202,36 +2202,45 @@ public void CopyToDestinationFolderWithSymbolicLinkCheck()
 
                 bool success = t.Execute();
 
-                Assert.True(success); // "success"
-                Assert.True(File.Exists(destFile)); // "destination exists"
-                Microsoft.Build.UnitTests.MockEngine.GetStringDelegate resourceDelegate = new Microsoft.Build.UnitTests.MockEngine.GetStringDelegate(AssemblyResources.GetString);
+                var needHigherPrivileges = me.Log.Contains("0x80070522");
 
-                me.AssertLogContainsMessageFromResource(resourceDelegate, "Copy.SymbolicLinkComment", sourceFile, destFile);
+                if (needHigherPrivileges)
```

Comment:
Yes, that'll work.

---
In reply to: [70453742](https://github.com/Microsoft/msbuild/pull/742#discussion_r70453742) [](ancestors = 70453742)